src/widgets/requirements/ui/requirements.module.scss

```diff
@@ -0,0 +1,30 @@
+.requirements {
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+}
+
+.title {
+  margin-bottom: 16px;
+}
+
+.title-margin {
+  margin-bottom: 16px;
+}
```

Comment:
I don't see this being used, maybe it should be deleted?

Reply:
Yes, thanks!
Done